target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -84,6 +84,9 @@ ath79_setup_interfaces()
 	etactica,eg200)
 		ucidef_set_interface_lan "eth0" "dhcp"
 		;;
+	engenius,ews511ap)
```

Comment:
Please keep the entry lexicographically sorted. engenius should be moved before etactica.